app/javascript/dashboard/components/widgets/WootWriter/FullEditor.vue

```diff
@@ -16,23 +23,31 @@ import {
   EditorState,
   Selection,
 } from '@chatwoot/prosemirror-schema';
-
+import { checkFileSizeLimit } from 'shared/helpers/FileHelper';
+import alertMixin from 'shared/mixins/alertMixin';
 import eventListenerMixins from 'shared/mixins/eventListenerMixins';
 import uiSettingsMixin from 'dashboard/mixins/uiSettings';
 
-const createState = (content, placeholder, plugins = []) => {
+const MAXIMUM_FILE_UPLOAD_SIZE = 16; // in MB
```

Comment:
Should we reduce this to 4 or 5 MB? Allowing these file sizes is not good for either our storage costs neither loading speed on their web pages
```suggestion
const MAXIMUM_FILE_UPLOAD_SIZE = 4; // in MB
```

Reply:
Makes sense